parser/SqlParser.ypp

```diff
@@ -759,6 +777,40 @@ opt_table_constraint_def_commalist:
     /* $$ = $1; */
   };
 
+opt_block_properties:
+  TOKEN_WITH TOKEN_BLOCKPROPERTIES '(' block_property_item_list ')' {
+    $$ = new quickstep::ParseBlockProperties(@1.first_line, @1.first_column, $4);
+  } | {
```

Comment:
For consistency with the style of cases in the rest of this file, put the `|` beginning the next case on its own line after the closing `}` bracket.